shell/components/form/KeyValue.vue

```diff
@@ -665,11 +659,14 @@ export default {
         </div>
       </template>
       <template
-        v-for="(row,i) in filteredRows"
+        v-for="(row,i) in rows"
         v-else
         :key="i"
       >
-        <div class="rowgroup">
+        <div
+          class="rowgroup"
+          :class="{'hidden': isProtected(row.key) && !toggleFilter}"
```

Comment:
We could replace the `hidden` class with the `hide` class. I figure we might as well use the global style since it exists.

https://github.com/rancher/dashboard/blob/master/shell/assets/styles/base/_helpers.scss#L134